robotino_controller_configuration_gazebo/package.xml

```diff
@@ -13,6 +13,8 @@
   <build_depend>roslaunch</build_depend>
   <build_depend>rostopic</build_depend>
   <build_depend>gazebo_ros_control</build_depend>
+  <build_depend>gazebo-plugins</build_depend>
```

Comment:
good question. I added that some month ago and as far as I can remember, I had to install those two packages manually to make it work. But maybe we don't need them. Should I remove the two packages?